File: Longest Common Prefix/Longest Common Prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        String start = strs[0];
        for(int i = 0; i < strs.length; i++){
            while(strs[i].indexOf(start) != 0){
                start = start.substring(0, start.length() - 1);
                if(start.isEmpty()) return "";
            }
        }
        return start;
    }
}

File: Longest Common Prefix/Longest Common Prefix.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        string start = strs[0];
        for (int i = 1; i < strs.size(); i++)
        {
            /* code */
            while (strs[i].find(start) != 0)
            {
                start = start.substr(0, start.size() - 1);
                if(start.empty()) return "";
            }
        }
        return start;
    }
};